packages/xstate-inspect/src/server.ts

```diff
@@ -73,6 +73,7 @@ export function inspect(options: ServerInspectorOptions): Inspector {
         return;
       }
 
+      // eslint-disable-next-line @typescript-eslint/no-base-to-string
       const jsonMessage = JSON.parse(data.toString());
```

Comment:
Would it be OK with this?
```suggestion
      const jsonMessage = JSON.parse(String(data));
```